documentation.json

```diff
@@ -0,0 +1,129 @@
+[
```

Comment:
Ah missed that one, I've got the elm-format beta installed and had to roll back a bunch of style changes.